pkg/controller/plan/util/utils.go

```diff
@@ -40,6 +40,15 @@ func CalculateSpaceWithOverhead(requestedSpace int64, volumeMode *core.Persisten
 	return spaceWithOverhead
 }
 
+func GetBootDiskNumber(deviceString string) int {
+	resp := GetDeviceNumber(deviceString)
```

Comment:
`resp` onse as a var name is not clear for me, maybe we can use a different name ?